### ISSUE TITLE
Do you support Windows system

### ISSUE THREAD

**Issue by echo-LuGuang** [kenshaw/escpos#19]
Do you support Windows system

**Comment by Andrew-1000** [comment]
> Do you support Windows system

@echo-LuGuang... did you manage to get a workaround for windows..?

**Comment by zuoliang0** [comment]
Can this library be used on Windows?

**Comment by kenshaw** [comment]
No, this can't work out of the box on Windows. There might be a way to figure out how to get it to work, using the right DLL, but it's not likely I'll be able to figure this out any time soon.
